definitions/npm/google-protobuf_v3.6.x/test_google-protobuf.js

```diff
@@ -0,0 +1,8 @@
+// @flow
+import * as jspb from "google-protobuf";
+
+var message = new jspb.Message();
+
+var map = new jspb.Map([["key", "value"]]);
+
+console.log(map.get("key") === "value")
```

Comment:
I guess this could use a bit more tests? 

Could you please wrap tests in a few [`describe/it` blocks](https://github.com/flow-typed/flow-typed/blob/master/CONTRIBUTING.md#use-describe-and-it-blocks-to-limit-scope)?